addons/mail/static/src/models/message_in_reply_to_view/message_in_reply_to_view.js

```diff
@@ -29,6 +31,13 @@ registerModel({
             }
             parentMessageView.update({ doHighlight: true });
         },
+        /**
+         * @private
+         * @returns {Markup}
+         */
+        _computeContentAsMarkup() {
+            return markup(this.messageInReplyToView.messageView.message.parentMessage.prettyBody);
+        },
```

Comment:
`markup` should be produced as close as possible to the original source of the "injection"
In this case, it should be directly into `prettyBody`, or worse case as a `prettyBodyMarkup` field.